docs/src/Box.doc.js

```diff
@@ -651,23 +803,38 @@ function ButtonFlyoutExample() {
     </>
   );
 }`}
-  />,
-);
-
-card(
-  <Example
-    description={`
-It's possible to use box with external elements using the css \`z-index\` property by capturing those values in controlled objects. The example below shows using a \`FixedZIndex\` for a value that comes from somewhere else, and a \`CompositeZIndex\` to layer the Box on top of it.
-  `}
-    id="zindex"
-    name="ZIndex"
-    defaultCode={`
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      description={`It's possible to use Box with external elements using the css \`z-index\` property by capturing those values in controlled objects. The example below shows using a \`FixedZIndex\` for a value that comes from somewhere else, and a \`CompositeZIndex\` to layer the Box on top of it.`}
+      title="Z-Index"
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
 function Example() {
   const HEADER_ZINDEX = new FixedZIndex(100);
   const zIndex = new CompositeZIndex([HEADER_ZINDEX]);
   return <Box color="blue" width={60} height={60} zIndex={zIndex} />
 }
 `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Related"
+    description={`
+      [Flex](/Flex)
+
+      - Use Flex and Flex.Item for flex layouts when you don't need the extra functionality in Box, and could benefit from the ability to supply a gap between items.
+
+      [Container](/Container)
+
+      - Use Container to responsively layout content at different screen sizes.
```

Comment:
This doesn't really say much…it's fine for now, but we should expand on this when we get to the Container docs